pkg/cmd/roachtest/tests/cdc.go

```diff
@@ -245,6 +252,27 @@ func (ct *cdcTester) setupSink(args feedArgs) string {
 		}
 
 		sinkURI = kafka.sinkURL(ct.ctx)
+	case azureEventHubKafkaSink:
+		kafkaNode := ct.kafkaSinkNode()
+		kafka := kafkaManager{
+			t:     ct.t,
+			c:     ct.cluster,
+			nodes: kafkaNode,
```

Comment:
Nit: might be more descriptive to call it `kafkaSinkNode`. Otherwise, at the callsite (of `RunWithDetailsSingleNode`) it's unclear whether it's a single-element slice.

Reply:
I agree with the confusion here. I was also bothered by it. I'm naming it as kafkaNode since this has been the [convention](https://github.com/cockroachdb/cockroach/blob/404376662fea066cb6f1968d5a1ecfb3c09ae467/pkg/cmd/roachtest/tests/cdc.go#L237) in cdc roachtests. But we should probably use the field name node rather than nodes since it is always a single node for kafka in roachtests/cdc. I will open another PR if you think this is a good idea.

Reply:
It's really a "nit", so it doesn't really merit another PR, imo. You could just `Refactor->Rename` here.

Reply:
I'm not sure if I'm interpreting the comment here right - why would renaming kafkaNode to kafkaSinkNode improve clarity regarding single-element slice. I thought it would make more sense to rename the field name `nodes` of kafkaManager to `node` to emphasize the singularity. Using kafkaNode seems to be the naming [convention](https://github.com/cockroachdb/cockroach/blob/404376662fea066cb6f1968d5a1ecfb3c09ae467/pkg/cmd/roachtest/tests/cdc.go) for all roachtests, so I thought just renaming it in this PR might lead to inconsistencies.

Reply:
> why would renaming kafkaNode to kafkaSinkNode

Indeed, that's not what the comment suggested, rather the `nodes` field which is plural.

Reply:
Ahh I see. Sorry for the misunderstanding here. I added the changes as a second commit as part of this PR.